dust_extinction/parameter_averages.py

```diff
@@ -9,13 +10,14 @@
 from .averages import G03_SMCBar
 from .shapes import _curve_F99_method
 
-__all__ = ['CCM89', 'O94', 'F99', 'F04', 'G16']
+__all__ = ['CCM89', 'O94', 'F99', 'F04', 'G16', 'MA14']
```

Comment:
Minor nit.  It would interchange the order of 'G16' and 'MA14' to keep them in chronological order.  And It should be 'M14' instead of 'MA14' to keep to the naming convention.

Reply:
Will do!